releaseNotes/Microsoft.Featuremanagement.md

```diff
@@ -2,11 +2,29 @@
 
 [Source code][source_code] | [Package (NuGet)][package] | [Samples][samples] | [Product documentation][docs]
 
+## 4.0.0-preview4 - Jul 19, 2024
+
+### Enhancements
+
+* Feature flags declared in the [.NET Feature Management schema](https://github.com/microsoft/FeatureManagement-Dotnet/blob/main/schemas/FeatureManagement.Dotnet.v1.0.0.schema.json) will also be recognized when they appears together with [Microsoft Feature Management schema](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureManagement.v2.0.0.schema.json). [#470](https://github.com/microsoft/FeatureManagement-Dotnet/pull/470)
+
+* Feature manager now publish telemetry through `Activity`. If telemetry is enabled for a feature flag, whenever the feature flag is evaluated, feature manager will start an `Activity` and add an `ActivityEvent` with tags contains feature evaluation information.  [#455](https://github.com/microsoft/FeatureManagement-Dotnet/pull/455)
```

Comment:
If someone wants to implement their own telemetry publisher, is there a particular activity they should listen to? I am wondering how much this is just implementation detail and what we should disclose.

Reply:
> If someone wants to implement their own telemetry publisher, is there a particular activity they should listen to?

Yes. The acitivity source with name "Microsoft.FeatureManagement" and version "1.0.0" should be listened.